DatabaseAccess/EmailSender/EmailSender.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
```

Comment:
fisierul asta nu cred ca ar trebui sa existe

(e in DatabaseAccess si nu prea are ce cauta acolo, are ca namespace BloodPlus.Services, mai e adaugat o data din ce vad in services/emailSender)